CONTRIBUTORS.md

```diff
@@ -265,3 +265,4 @@
  - [0x25CBFC4F](https://github.com/0x25CBFC4F)
  - [Robert Lützner](https://github.com/rluetzner)
  - [Nathan McCrina](https://github.com/nfmccrina)
+ - [benedikt257](https://github.com/benedikt257)
```

Comment:
This is on the wrong place, add it to the list of Jellyfin contributors instead of Emby

Reply:
Sorry, didn´t see that there were two sections here. Should be fixed now, many thanks for spotting!